CHANGELOG.md

```diff
@@ -11,6 +11,10 @@ incremented for features.
 
 ## [Unreleased]
 
+### Breaking Changes
+
+* lang: Mutate instead return new one in `CpiAccount::reload` ([#526](https://github.com/project-serum/anchor/pull/526)).
```

Comment:
```suggestion
* lang: `CpiAccount::reload` mutates the existing struct instead of returning a new one ([#526](https://github.com/project-serum/anchor/pull/526)).
```